CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
 
+## [v2.3.0] - 2021-04-28
+
+- adds update-profile step to the Automations API
```

Comment:
Can you add the function name that is being updated? This library wraps the API. I'd like to be a little more specific here.

Reply:
Yep 💯 - done